docs/tutorial.md

```diff
@@ -441,12 +441,14 @@ Working copy now at: mrxqplyk 52a6c7fd ABCD
 Parent commit      : kwtuwqnm 643061ac ABC
 ```
 
-That will bring up the built-in diff editor with a diff of the changes in
-the "ABCD" commit. If you prefer another diff editor, you can configure
-[`ui.diff-editor`](config.md#editing-diffs) instead. Modify the right side of
-the diff to have the desired end state in "ABC" by removing the "D" line. Then
-save the changes and close Meld. If we look at the diff of the second commit, we
-now see that all three lines got capitalized:
+That will bring up the built-in diff editor with a diff of the changes in the
+"ABCD" commit. If you prefer another diff editor, such as
+[Meld](https://meldmerge.org), you can configure
+[`ui.diff-editor`](config.md#editing-diffs) instead. Expand the file by clicking
+on `(+)` or with right arrow, then select the sections/line to include by double
```

Comment:
comment (non-blocking): Right arrow is technically not the "expand" operation; it does something context-sensitive. I personally use `f`/`F` to expand immediately without moving the cursor. No change needed here, since the arrow keys are probably the best advice to give newcomers.

comment: I don't think double-click does anything differently than single-click? I might misremember.

Reply:
That makes sense with me. I'm open to suggestions on how to write this better.

Changed "double clicking" to just "clicking". The first click was to select, the second to include.

Reply:
@arxanas Updated copy to include a note on clicking the menu.